scripts/collections.js

```diff
@@ -28,24 +30,54 @@ export function getAssetIdFromCollectionItem(assetItem) {
   return assetItem.id;
 }
 
+async function getRequestHeadersSearchCollections() {
+  const token = await getBearerToken();
+  return {
+    'Content-Type': 'application/json',
+    'x-api-key': 'asset_search_service',
```

Comment:
Could you please use getBackendApiKey instead of hard coding it

Reply:
Replaced with 'x-api-key': getBackendApiKey(),